src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminCourtAddressControllerTest.java

```diff
@@ -95,4 +101,20 @@ void shouldReturnNotFoundWhenUpdatingAddressesForUnknownCourtSlug() throws Excep
             .andExpect(status().isNotFound())
             .andExpect(content().string("Not found: " + TEST_SLUG));
     }
+
+    @Test
+    void shouldReturnBadRequestWhenUpdatingAddressesWithAnInvalidPostcode() throws Exception {
```

Comment:
Should we also have a test for if mapit returns with a no result for a postcode search? I.e {} as a response, indicating a 404? Or a not found?

Reply:
Yeah can add a test for that